### PR TITLE
Fixed docs typo in production-checklist.rst: --dry-dun -> dru-run

### PR DESCRIPTION
**Pull request by alisher-matkurbanov**
Fixed docs typo in production-checklist.rst
>> but will pass with ``--dry-dun`` option
Fixed to:
>> but will pass with ``--dry-run`` option.

### CODE REVIEW

File: {{cookiecutter.project_name}}/docs/pages/template/testing.rst [hunk 1]
```diff
@@ -108,4 +108,4 @@ for more information.
 We also use `django-stubs <https://github.com/typeddjango/django-stubs>`_
 to type ``django`` internals.
 This package is optional and can be removed,
-if you don't what to type your ``django`` for some reason.
+if you don't want to type your ``django`` for some reason.
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docs/pages/template/testing.rst]: This change is unrelated, please remove it

Comment (alisher-matkurbanov) [reply {{cookiecutter.project_name}}/docs/pages/template/testing.rst]: I'm sorry, how can I remove unrelated commit?

Comment (sobolevn) [reply {{cookiecutter.project_name}}/docs/pages/template/testing.rst]: It is easier to open a new PR in this case 🙂
